base/cypress.md

```diff
@@ -8,13 +8,28 @@ This tool is integrated to RERO-ILS and the following documentation explains how
 
 * `poetry run bootstrap`
 * `poetry run setup`
-* `poetry run server`
+* `FLASK_DEBUG=False poetry run server`
```

Comment:
I may be wrong, but shouldn't it be:
```suggestion
* `export FLASK_DEBUG=False`
* `poetry run server`
```

Reply:
No. Really. If you export this variable and want to launch the server in development mode with debug toolbar, it will never appears.
It's really more careful to use it in the same command line!

Reply:
I didn't know it was even possible. Sorry. :slightly_smiling_face: